components/movementsensor/mpu6050/mpu6050_test.go

```diff
@@ -17,43 +17,27 @@ import (
 )
 
 func TestValidateConfig(t *testing.T) {
-	boardName := "local"
-	t.Run("fails with no board supplied", func(t *testing.T) {
-		cfg := Config{
-			I2cBus: "thing",
-		}
-		deps, err := cfg.Validate("path")
-		expectedErr := utils.NewConfigValidationFieldRequiredError("path", "board")
-		test.That(t, err, test.ShouldBeError, expectedErr)
-		test.That(t, deps, test.ShouldBeEmpty)
-	})
-
 	t.Run("fails with no I2C bus", func(t *testing.T) {
-		cfg := Config{
-			BoardName: boardName,
-		}
+		cfg := Config{}
 		deps, err := cfg.Validate("path")
 		expectedErr := utils.NewConfigValidationFieldRequiredError("path", "i2c_bus")
 		test.That(t, err, test.ShouldBeError, expectedErr)
 		test.That(t, deps, test.ShouldBeEmpty)
 	})
 
-	t.Run("adds board name to dependencies on success", func(t *testing.T) {
+	t.Run("no dependencies on success", func(t *testing.T) {
```

Comment:
[q] do we even want this test? Seems like it's not testing anything important anymore. If the board dependency is never used, why does it matter whether it is in the dependencies or not?

Reply:
Good point, I'll remove it!